docs/IE_PLUGIN_DG/ExecutableNetwork.md

```diff
@@ -1,7 +1,7 @@
 # Executable Network {#executable_network}
 
 `ExecutableNetwork` class functionality:
-- Compile an InferenceEngine::ICNNNetwork instance to a hardware-specific graph representation
+- Compile an InferenceEngine::ICNNNetwork instance to a backend-specific graph representation
```

Comment:
plugin-specific? 

In IE terminology we have CPU, GPU, VPU plugins (not backends).

Reply:
I just want to separate plugin and backend (while it's mixed for now). Plugin is a wrapper around backend, while backend is MKLDNN, CLDNN, CUDA, ARMNN libraries, etc. And in this case graph is backend specific.

How do you think?

Reply:
OK. Is it mentioned somewhere?

Reply:
I will check plugin documentation and mention it in plugin description.

Reply:
added more details documentation for plugin describing that it's a wrapper around a backend